src/measure_theory/tactic.lean

```diff
@@ -124,11 +124,13 @@ do t ← tactic.target,
   | _ := skip
   end
 
-/-- List of tactics used by `measurability` internally. -/
+/-- List of tactics used by `measurability` internally. The option `use_exfalso := ff` is passed to
+the tactic `apply_assumption` in order to avoid loops in the presence of negated hypotheses in
+the context. -/
 meta def measurability_tactics (md : transparency := semireducible) : list (tactic string) :=
 [
-  propositional_goal >> apply_assumption
-                        >> pure "apply_assumption",
+  propositional_goal >> tactic.interactive.apply_assumption none {use_exfalso := ff}
+                        >> pure "tactic.interactive.apply_assumption none {use_exfalso := ff}",
```

Comment:
Should it print `tactic.interactive` in the suggestion?

Reply:
Sorry, but I understand what I am doing so little that I don't even known what here has to do with printing. I also can't get an example in which `apply_assumption` would be suggested by the tactic. Take the following example:
```
example {α β : Type*} [measurable_space α] [measurable_space β]
  {f : α → β} {p : (α → β) → Prop} (hp : p f) (h : ∀ g, p g → measurable g) :
  measurable f :=
by measurability?
```
The suggestion is `exact h f hp`. If I don't put `hp` in the hypotheses, I get a "try this: `refine h f _`". There is no `apply_assumption` appearing in those.

In summary: I don't know what I am doing and I would accept any help and suggestion. If you think that I should remove `tactic.interactive` from line 133, I'll do it.